hera/nfs/html_modifier.py

```diff
@@ -0,0 +1,50 @@
+from hera.workflows import WorkflowTemplate, DAG, Container, Task, Resources, models as m
```

Comment:
I don't know what's failing in here, but it's very upset in the logs:

```
/usr/bin/sh: 1: apk: not found
Start Clone of source branch
/usr/bin/sh: 5: git: not found
/usr/bin/sh: 9: git: not found
/usr/bin/sh: 10: git: not found
/usr/bin/sh: 11: git: not found
/usr/bin/sh: 13: git: not found
Merge in target branch
/usr/bin/sh: 16: git: not found
Complete.



```

Reply:
I don't actually see a `sed` or similar going on here? Is this a copy/paste of git checkout and alan got confused?

Reply:
Yes.